### PR TITLE
adding my project

### PR DESCRIPTION
**Pull request by vesellov**
New innovative HI-tech project is waiting for your skills, inspiration and commitment. Join BitDust team and be a part of a great open source development!

BitDust is looking for a Python developers to work on a new encrypted mobile messaging & file storage APP. This will be your chance to learn new technologies, have fun coding together and shine on the open source front.

BitDust APP also requires amazing UI and we are looking for a super-creative WEB designers and front-end developers to create wonderful user experience in the APP.

Our aim is to provide a healthy alternative to "centralized" social apps driven by large enterprise organizations. We are building a social software that actually belongs to people and is driven by community. We believe your data must be yours and you should be in full control of it.

We currently have a working Alpha version and are looking to increase the stability of the platform and add more features.

Would like to invite every enthusiastic developer to cooperate or a newbie who wish to learn new stuff.

Hope it fits Up for Grabs.

### CODE REVIEW

File: _data/projects/bitdust.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: BitDust
+desc: BitDust APP provides independent P2P on-line storage and secure private messaging
+site: https://bitdust.io
+tags:
+  - python
+  - git
```
Comment (shiftkey) [inline _data/projects/bitdust.yml]: For most projects listed on the site we recommend 5 or so good labels - could this be trimmed down to the tags that are important to the project?

**Review by shiftkey — COMMENTED** [review]
@vesellov thanks for the contribution!

Just one suggestion about the labels, everything else is :gem:

File: _data/projects/bitdust.yml (hunk 1 above)
Comment (vesellov) [reply _data/projects/bitdust.yml]: sure! I put 5 tags. Thanks @shiftkey

Comment (vesellov) [reply _data/projects/bitdust.yml]: 👍

**Review by shiftkey — APPROVED** [review]

**Comment by vesellov** [comment]
Thanks 👍

**Pull request merged**